client/test/models/article.test.coffee

```diff
@@ -89,11 +89,6 @@ describe "Article", ->
 
   describe '#getPublishDate', ->
 
-    it 'returns the current date if unpublished and no scheduled_publish_at', ->
-      @article.set published: false
-      now = moment().format('YYYY-MM-DD')
-      @article.getPublishDate().should.containEql now
-
```

Comment:
Was this removed intentionally?

Reply:
Yeah cause it was failing and I couldn't get it to pass initially but it works now so I added it back